capstone/src/main/java/com/google/univiz/CarnegieSizeDegree.java

```diff
@@ -26,8 +26,13 @@
   SIZE18("Exclusively graduate and professial institution");
 
   private final String degree;
+  private static CarnegieSizeDegree[] degreesList = CarnegieSizeDegree.values();
 
   CarnegieSizeDegree(String degree) {
     this.degree = degree;
   }
+
+  public static CarnegieSizeDegree getDegree(int index) {
+    return degreesList[index].degree;
```

Comment:
There's a mismatch here between the `String` and the return type of this method.